README.md

```diff
@@ -79,6 +79,18 @@ You must install [Msys2](https://www.msys2.org/) first, then **remember choosing
 
 The **default MSYS environment** is based on Cygwin, which **won't work**!
 
+*steps in order to build for windows:
+-Download msys2 then open the shell and type those commands respectively:
+ update pacman -Syu then pacman -su.
+-Open msys2-mingw64 shell from the start menu.
+-Type these three commands: 
+ pacman -s  
+ pacman --needed -s git mingw-w64-x86-gcc base-devel
+ pacman mingw-w64-x86-toolchain
```

Comment:
You can use this technique to make those three lines of code look nicer: https://www.markdownguide.org/basic-syntax/#code-blocks-1

(TL;DR: indent those lines with 8 spaces)